wrapper:
  config_file: avia.yaml
  lidar_topic: /livox/lidar
  imu_topic: /livox/imu
  lidar_type: 0
  
sectional_lio:
  filter_leaf_size: 0.5
  extrin_r: [1,0,0,
            0,1,0,
            0,0,1]
  extrin_t: [0.4165,0.2326,-0.2840]
  section_min_map_size: 13000
eskf:
  cov_gyroscope: 0.1
  cov_acceleration: 0.1
  cov_bias_acceleration: 0.0001
  cov_bias_gyroscope: 0.0001

section_process:
  max_sections: 3
  planaer_threhold: 0.2
  section_min_point_size: 1800
